tests/unit/packaging/test_views.py

```diff
@@ -186,7 +233,7 @@ def test_detail_rendered(self, db_request):
                 description=DescriptionFactory.create(
                     raw="unrendered description",
                     html="rendered description",
-                    content_type="text/plain",
+                    content_type="text/html",
```

Comment:
these changes are to make the behavior track the common use case - `text/html`.
I added a similar test for plain above these two - there wasn't enough overlap for the existing `_rendered/_renders` tests to refactor to a parameterize, but can do that if desired.